sdks/java/io/debezium/src/main/java/org/apache/beam/io/debezium/DebeziumTransformRegistrar.java

```diff
@@ -34,7 +34,7 @@
   "nullness" // TODO(https://github.com/apache/beam/issues/20497)
 })
 public class DebeziumTransformRegistrar implements ExternalTransformRegistrar {
-  public static final String READ_JSON_URN = "beam:transform:org.apache.beam:debezium_read:v1";
+  public static final String READ_JSON_URN = "beam:transform:org.apache.beam:debezium_read:v3";
```

Comment:
This "v1" is a URN for Beam transform. Change the name only in Java will break Python transform. No need to change

Reply:
Thanks for that catch!